### ISSUE TITLE
I want to buy you a beer, but I can't

### ISSUE THREAD

**Issue by martinth** [ptsochantaris/trailer#363]
Hey there.

I really love Trailer and introduced it so multiple of my colleagues and it has really improved our workflow. I noticed that their is no "Donate" option or anything like that.

I just want to show my appreciation for your great work and I feel a simple "thank you" isn't enough. 

P.S.: Doesn't have to be beer ;)

**Comment by ptsochantaris** [comment]
Hi @martinth and thank you so much for your nice comments, I really appreciate them, and I'm really happy to know that Trailer is making your life better to whatever degree it can! I try to keep an eye out for suggestions and/or new features, so feel free to post ideas on how to improve it, if you have any.

On the issue of a donate button, I have considered it, but ultimately decided that the hassle (and possible cost) of setting up the paperwork for that would probably outweigh any donations that would come my way - not that some people wouldn't gladly offer some tips, it just... well, I think I'm privileged enough as it is, and have managed to build good things because of the hard work of others anyway, so I'm glad I can put something back out there in turn. Hearing from people that it's helping them do better work is a huge deal for me. Don't get me wrong - if I thought I'd be making serious money from Trailer somehow, I'd do it in a heartbeat :D but if I were to chose between some small donations as payment for working on Trailer vs. viewing Trailer as my own contribution back to the community, well, I'll forego the small financial amounts and take the good karma instead :)

That said, if you really can't help but want to give me money then feel free to look into my commercial app, Gladys, which is available for both [macOS](https://itunes.apple.com/us/app/gladys/id1382386877) and [iOS](https://itunes.apple.com/us/app/gladys/id1257526927), is quite cheap, and can be useful when developing too (for instance, storing and syncing code snippets, links, docs, data files, etc) - If it doesn't tickle your fancy, then I'll still take the beer at some point :D 

In closing, thanks again for your very kind comments, they really do make my day, and enjoy Trailer!

**Comment by WhiteBoardDev** [comment]
I installed trailer yesterday and it already cut down a lot of communication overhead about changes to PRs etc. Thank you for building this! Hopefully I can get some time to contribute back to the repo sometime as a thank you.

**Comment by ptsochantaris** [comment]
Hi guys and thank you so much for your kind comments. I'm really glad Trailer is being useful for you. Any and all contributions are very welcome indeed. Development pace on Trailer has slowed a bit from my end recently, as work this past year has been keeping me very busy, but my TODO list, based on all the great suggestions and requests that show up in this repo, is still growing, and I hope to start catching up with it, as I've fallen behind :) That said, any and all contributions and suggestions continue to be more than welcome!

@stembrain Gladys is not quite the same as Jumpcut, it's more like a shelf/container where you drag things to keep them temporarily, rather than drag items from one app to another. I made it specifically for iOS 11 iPad users, but the app has since expanded to serve iPhone users and macOS users too - optionally syncing the shelf state between them via iCloud. It's a weird Swiss-army knife of an app :) since it can act as a clipboard manager, but can also be a data investigation tool, an indirect replacement for Airdrop, or a very simple ultra-tiny-file manager :) However, it is less about the clipboard per-se and more about enabling smoother workflows on your iPad and/or peripheral devices.

@WhiteBoardDev If you do decide to dive in, welcome!, and I can only apologise for the state of the code, as this project grew out of a tiny in-house utility to help co-workers with PRs :) but I'm more than happy to answer any questions you may have related to the code and architecture, in case you feel like contributing! As a new user, any feedback or thoughts you have on the app would definitely be very valuable, so don't be shy about opening issues with those.

**Comment by pvinis** [comment]
What about something simple like https://www.buymeacoffee.com/ ?

**Comment by ptsochantaris** [comment]
Hi! As much as I love to be above it all, I've decided that I hate an open bug issue even more, so I've fixed this, by putting up a sponsorship link on my repos.

The reasoning behind this is so I can give myself more excuses to work on these open source projects vs my commercial work, or at least pay for the unhealthily soft drinks consumed in the course of said work 😄

Huge thanks to anyone wishing to contribute, and as importantly, absolutely fine to not contribute: Trailer will always be free to use.
